clang/docs/ReleaseNotes.rst

```diff
@@ -327,6 +327,9 @@ Non-comprehensive list of changes in this release
   ``__reference_constructs_from_temporary`` should be used instead. (#GH44056)
 - Added `__builtin_get_vtable_pointer` to directly load the primary vtable pointer from a
   polymorphic object.
+- `libclang` receives a family of new bindings to query basic facts about
+  GCC-style inline assembly blocks, including whether the block is `volatile`
+  and its template string following the LLVM IR `asm` format. (#GH143424)
```

Comment:
```suggestion
- ``libclang`` receives a family of new bindings to query basic facts about
  GCC-style inline assembly blocks, including whether the block is ``volatile``
  and its template string following the LLVM IR ``asm`` format. (#GH143424)
```
Adding some more backticks because RST is funny sometimes.